session/pingpong/hermes_promise_settler.go

```diff
@@ -423,6 +423,13 @@ func (aps *hermesPromiseSettler) settle(
 				log.Error().Err(err).Msg("Could not generate provider channel address")
 			}
 
+			ch, err := aps.channelProvider.Fetch(provider, hermesID)
```

Comment:
This `TotalSettled ` is not actual from customer perspective, suggest to avoid this external call and remove it.

Reply:
ok, provider balance is renewed this this, correct?